src/module/add.js

```diff
@@ -1,22 +1,21 @@
-import renderToDo from './display.js';
+// import { UI } from './storeData.js'
+// export const renderDomContentDb = () => {
+//     const newData = JSON.parse(localStorage.getItem('data') || '[]');
 
-const form = document.getElementById('form');
+//     form.addEventListener('submit', (e) => {
+//       e.preventDefault();
+//       const InputValue = document.querySelector('#input-list').value;
 
-const addToDo = (e) => {
-  const newData = JSON.parse(localStorage.getItem('data') || '[]');
-  e.preventDefault();
-  const InputValue = document.querySelector('#input-list').value;
+//       const dataObj = {
+//         description: InputValue,
+//         completed: false,
+//         index: newData.length,
+//       };
```

Comment:
- [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 🙏🏼